entries/u/ukraine.com.ua.json

```diff
@@ -10,7 +10,7 @@
     "custom-software": [
       "Telegram Bot"
     ],
-    "documentation": "https://www.ukraine.com.ua/en/wiki/account/security/otp/",
+    "documentation": "https://www.ukraine.com.ua/uk/wiki/account/security/otp/",
```

Comment:
I'm also getting a timeout error here.

Reply:
Loads for me but the content is in Ukranian.

Reply:
The English pages seem to be gone. :disappointed:

Reply:
Should we set it to the Ukranian version then?

Reply:
What's our course of action then?

Reply:
There are many doc pages that are not in English, and nothing says they have to be.

Reply:
Should we keep this URL or remove the language code from it?

Reply:
done